README.rst

```diff
@@ -215,31 +215,63 @@ Other Python wrappers for GMT:
 * `PyGMT <https://github.com/glimmer-cism/PyGMT>`__  by `Magnus Hagdorn <https://github.com/mhagdorn>`__
 
 
-Documentation for other versions
---------------------------------
-
-* `Development <https://www.pygmt.org/dev>`__ (reflects the *main* branch on
-  GitHub)
-* `Latest release <https://www.pygmt.org/latest>`__
-* `v0.4.0 <https://www.pygmt.org/v0.4.0>`__
-* `v0.3.1 <https://www.pygmt.org/v0.3.1>`__
-* `v0.3.0 <https://www.pygmt.org/v0.3.0>`__
-* `v0.2.1 <https://www.pygmt.org/v0.2.1>`__
-* `v0.2.0 <https://www.pygmt.org/v0.2.0>`__
-* `v0.1.2 <https://www.pygmt.org/v0.1.2>`__
-* `v0.1.1 <https://www.pygmt.org/v0.1.1>`__
-* `v0.1.0 <https://www.pygmt.org/v0.1.0>`__
-* `v0.0.1a0 <https://www.pygmt.org/0.0.1a0>`__
-
-Compatibility with GMT and Python/NumPy versions
-------------------------------------------------
-
-======= ========== ========= =========
-PyGMT   GMT        Python    NumPy
-======= ========== ========= =========
-0.4.x   >=6.2.0    >=3.7     >=1.17.0
-0.3.x   >=6.1.1    >=3.7
-0.2.1   >=6.1.1    >=3.6
-0.2.0   >=6.1.1    3.6 - 3.8
-0.1.x   >=6.0.0    3.6 - 3.8
-======= ========== ========= =========
+Compatibility with GMT/Python/NumPy versions
+--------------------------------------------
+.. list-table::
+    :widths: 15 25 30 15 20 15
+    :header-rows: 1
+
+    * - PyGMT
+      - GitHub Release
```

Comment:
Maybe merge the PyGMT and GitHub Release columns?

```suggestion
    * - PyGMT
```

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/1415/commits/b02134733ba4cb4183b3f491ac439c9d468f5e1a